tests/testthat/_snaps/harmonic.md

```diff
@@ -51,7 +56,7 @@
     Condition
       Error in `step_harmonic()`:
       Caused by error in `bake()`:
-      ! variable must have at least one non-NA value
+      ! Variable must have at least one non-NA value.
```

Comment:
Would it be helpful to call out which variable is being referred to here?

Reply:
it would, but it requires a larger refactor that is too large for this PR and will be done later